include/openspace/events/event.h

```diff
@@ -397,6 +400,32 @@ struct EventPointSpacecraft : public Event {
     const double duration;
 };
 
+/**
+ * This event is created whenever a renderable is enabled.  By the
+ * time this event is signalled, the renderable has already been enabled.
+ *
+ * \param Node The identifier of the node that contains the renderable
```

Comment:
The \param line would need to be moved to the constructor.  Since this is a non-templated struct, it doesn't have any parameters

Reply:
Oh yea, just seeing that now..  someone (read: me) put the comments in the wrong place. Having it like:
```cpp
/**
 * This event is created when a screenspace renderable has been removed from the system.
 * When this event is created, the screenspace renderable has already been removed and is
 * no longer available
 */
struct EventScreenSpaceRenderableRemoved : public Event {
    static const Type Type = Event::Type::ScreenSpaceRenderableRemoved;

    /**
     *\param renderable_ The identifier of the screenspace renderable that was removed
     */
    explicit EventScreenSpaceRenderableRemoved(const ScreenSpaceRenderable* renderable_);
    const tstring renderable;
};
```

would generate better Doxygen comments for the vents